packages/stateManager/tests/mockClient.ts

```diff
@@ -144,10 +143,21 @@ export class MockClient implements IRelayClient {
     return { url: file.name };
   }
 
-  async commitOrder(order: schema.CommitItemsToOrderRequest, orderId: number) {
+  async commitOrder(
+    order: schema.CommitItemsToOrderRequest,
+    orderId: Uint8Array,
+  ) {
+    const eId = eventId();
     this.sendShopEvent({
+      updateOrder: { id: orderId, commit: order, eventId: eId },
+    });
+    return eId;
+  }
+  //Mimics client-fired event paymentDetails after commit event - for testing paymentDetails gets stored correctly in stateManager.
+  async sendPaymentDetails(orderId: `0x${string}`) {
+    return this.sendShopEvent({
       updateOrder: {
-        id: orderId,
+        id: hexToBytes(orderId),
         eventId: hexToBytes(
           "0x32b36377007de4ab0fcc3eabb1ef3a7096c42004c14babc3638f81b9d0982625",
         ),
```

Comment:
```suggestion
```

eventId's are not thing anymore. Is probably ignored and thus not a real issue but might be confusing to stumble over in the future.

Reply:
I know the relay ignores it but I am including eventId for the eventListenAndResolve function in stateManager. We could use id instead of eventId for the statemanager to listen and resolve, but in the edge case that two people update the same tag at the same time,  we don't want the update function to resolve from somebody else's update